common.blocks/sorting/sorting.js

```diff
@@ -0,0 +1,38 @@
+modules.define('sorting', ['i-bem__dom', 'select', 'jquery'], function(provide, BEMDOM, Select, $) {
+  provide(BEMDOM.decl(this.name, {
+    onSetMod: {
+      js: {
+        'inited': function() {
+          Select.on(this.elem('select'), 'change', this._onSortingSelectChanged, this);
+        }
+      }
+    },
+    _onSortingSelectChanged: function(e) {
+      var queryString = this._getQueryString(e.target.getVal()),
+      contentBlock = this.findBlockOutside('content'),
```

Comment:
looking for outer blocks is not quite good architecture.
it's better to trigger an event and to subscribe to it in outer block. so that child will never know about its parent.